### PR TITLE
042_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_non-media_app_RPC_…

### PR DESCRIPTION
**Pull request by istoimenova**
Script is part of TS: #932

### CODE REVIEW

**Comment by istoimenova** [comment]
@OVikhrov, @dboltovskyi, please review.

**Comment by dboltovskyi** [comment]
@istoimenova, Since this script is almost the same as 041 review is on Hold until review of 041 is finished

File: test_scripts/API/SetAudioStreamingIndicator/042_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_non-media_app_RPC_omitted_PTgroups.lua [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+--
+-- Description:
+-- In case the successfully registered application sends an RPC that is NOT included
+-- (omitted) in the PolicyTable group(s) assigned to the application
+-- SDL must: return DISALLOWED, success:false to this mobile app
+--
+-- 1. Used preconditions
+-- Assign to non-media application BASE-4 functional group.
+-- Remove from BASE-4 RPC SetAudioStreamingIndicator
+-- Register media application
```
Comment (dboltovskyi) [inline test_scripts/API/SetAudioStreamingIndicator/042_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_non-media_app_RPC_omitted_PTgroups.lua]: @istoimenova, Please fix: 'media' => 'non-media'

Comment (istoimenova) [inline test_scripts/API/SetAudioStreamingIndicator/042_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_non-media_app_RPC_omitted_PTgroups.lua]: @dboltovskyi, done in 0b08710a080580e4dc719651549e3de7fb04f7e8

File: test_scripts/API/SetAudioStreamingIndicator/042_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_non-media_app_RPC_omitted_PTgroups.lua [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+commonSteps:DeleteLogsFiles()
+commonSteps:DeletePolicyTable()
+
+--[[ General Settings for configuration ]]
+Test = require('connecttest')
+require('user_modules/AppTypes')
+
+--[[ Test ]]
+commonFunctions:newTestCasesGroup("Test")
+
```
Comment (OVikhrov) [inline test_scripts/API/SetAudioStreamingIndicator/042_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_non-media_app_RPC_omitted_PTgroups.lua]: @istoimenova Please activate and consent device and then send request. Currently you get DISALLOWED becouse device not consented

**Pull request merged**